packages/components/pie-notification/src/index.ts

```diff
@@ -84,19 +81,9 @@ export class PieNotification extends LitElement implements NotificationProps {
     @state()
     protected _hasIconClass = false;
 
-    @state()
-    protected _hasContentGutter = false;
```

Comment:
the behavior should be applied for default, thus, removing it and using it directly.